app/models/proxy_deposit_request.rb

```diff
@@ -31,55 +37,74 @@ def sending_user_should_not_be_receiving_user
   end
 
   def should_not_be_already_part_of_a_transfer
-    transfers = ProxyDepositRequest.where(work_id: work_id, status: 'pending')
+    transfers = ProxyDepositRequest.where(work_id: work_id, status: PENDING)
     errors.add(:open_transfer, 'must close open transfer on the work before creating a new one') unless transfers.blank? || (transfers.count == 1 && transfers[0].id == id)
   end
 
   def send_request_transfer_message
     if updated_at == created_at
+      send_request_transfer_message_as_part_of_create
+    else
+      send_request_transfer_message_as_part_of_update
+    end
+  end
+
+  private
+
+    def send_request_transfer_message_as_part_of_create
       user_link = link_to(sending_user.name, Hyrax::Engine.routes.url_helpers.profile_path(sending_user.user_key))
       transfer_link = link_to('transfer requests', Hyrax::Engine.routes.url_helpers.transfers_path)
       message = "#{user_link} wants to transfer a work to you. Review all #{transfer_link}"
       User.batch_user.send_message(receiving_user, message, "Ownership Change Request")
-    else
+    end
+
+    def send_request_transfer_message_as_part_of_update
       message = "Your transfer request was #{status}."
       message += " Comments: #{receiver_comment}" unless receiver_comment.blank?
       User.batch_user.send_message(sending_user, message, "Ownership Change #{status}")
     end
-  end
 
-  def pending?
-    status == 'pending'
-  end
+  public
 
-  def accepted?
-    status == 'accepted'
-  end
+  ACCEPTED = 'accepted'.freeze
+  PENDING = 'pending'.freeze
+  CANCELED = 'canceled'.freeze
+  REJECTED = 'rejected'.freeze
+
+  enum(
```

Comment:
@jcoyne I added a verifying test that we are indeed persisting a human readable string (instead of the integer as the documentation says it supports).